lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -1,6 +1,15 @@
 <template>
-	<div class="badge" :class="{ '-neutral': color === BADGE_COLORS.NEUTRAL }">
-		<div class="badge__slot">
+	<div
+		class="badge"
+		:class="{
+			'-neutral': color === BADGE_COLORS.NEUTRAL,
+			'-primary': color === BADGE_COLORS.PRIMARY,
+		}"
+	>
+		<div v-if="icon" class="badge__icon">
```

Comment:
why do we need a `div` wrapper? Can't we use `icon` directly?

Reply:
Sure